specification/network/resource-manager/readme.md

```diff
@@ -1601,6 +1602,9 @@ directive:
   - suppress: RequiredPropertiesMissingInResourceModel
     from: ipAllocation.json
     reason: name, id and type properties are inherited from the upper level
+    suppress: RequiredPropertiesMissingInResourceModel
```

Comment:
Here is reason for error duplicate key. you should put - suppress:    "-" is missed.
But I noticed there is huge suppression for lint error which we may need to review whether it is still need. as some of them maybe temporary

Reply:
By now this should be fixed. I was just being silly here.